bridge/src/benchmarking.rs

```diff
@@ -244,4 +244,84 @@ mod benchmarks {
 		assert_eq!(Balances::<T, _>::free_balance(caller), 200000000.into());
 		assert_eq!(Balances::<T, _>::free_balance(bridge_account), (amount - 200000000).into());
 	}
+
+	#[benchmark]
+	fn pause_all_bridges() {
```

Comment:
I see what you meant, to my understanding, this extrinsic is accessing two storages(DestDomainIDs, IsPaused), so I think the benchmark might need to take two length as parameters of each one to perform. And, the size of `IsPaused` is dynamically changing while extrinsic executing because what's in the `DestDomainIDs` will be topped up onto `IsPaused`, so it makes the situation even more complicated. 

Also as I'm looking into it, I found that there is no way to get the size of a `StorageMap` ([parity doc of StorageMap](https://paritytech.github.io/substrate/master/frame_support/storage/trait.StorageMap.html#)). Not sure if I'm missing anything here.

in summary, my idea is:
1. IF there is a way to get storageMap size, we add it as the benchmark parameter but only add size of `DestDomainIDs`. This way we will at least simulate the first storage loop logic to make benchmark more accurate, `IsPaused` in the most case will be the subset of the `DestDomainIDs` so we should be fine.
2. IF there is no way to get storageMap size, we hardcode an estimated size in the benchmark, for instance `2`, currently we have 2 domains on mainnet, we can update this number when we adding much more domains later, but for sure this wouldn't be a good methodology.

What you think? @tolak

Reply:
I also haven't found a way to get the storage size as the argument passes to the benchmark function. Maybe we can implement a dedicated method in pallet but I don't think it's a wise choice because every time ppl call the method it will iterator the map in advance.
As you suggested, let's just hardcode a reasonable number until we find a better solution